core/src/main/java/org/kiwix/kiwixmobile/core/base/BaseActivity.kt

```diff
@@ -36,16 +37,14 @@ abstract class BaseActivity : AppCompatActivity() {
 
   private var unbinder: Unbinder? = null
 
+  protected abstract fun injection(coreComponent: CoreComponent)
+
   override fun onCreate(savedInstanceState: Bundle?) {
-    injection()
+    injection(CoreApp.getCoreComponent())
```

Comment:
when you were adding this did you rewrite the function first? There is a couple ways to do this so the change propagates to all the call sites

just add it to get an error
![image](https://user-images.githubusercontent.com/3358282/72593755-b2733f00-38fd-11ea-9cb8-7e705f21f070.png)
and we want to add a parameter
![image](https://user-images.githubusercontent.com/3358282/72593819-dd5d9300-38fd-11ea-944c-007642dadfec.png)
that gives us the ChangeSignature modal and we just have to hit the refactor button (or alt+r) and all existing callsites are modified
as I can see in my list of changed files
![image](https://user-images.githubusercontent.com/3358282/72594137-a50a8480-38fe-11ea-921f-6facaf4ea766.png)

Alternatively you can just go straight to the ChangeSignature modal by right clicking on a method>Refactor>change signature, or placing your cursor on a method and pressing ctrl+f6